package.json

```diff
@@ -22,6 +22,7 @@
     "react": "18.1.0",
     "react-dom": "18.1.0",
     "react-hook-form": "^7.31.1",
+    "storybook-addon-next-router": "^4.0.0",
```

Comment:
Is there a special reason, that you did not install this addOn as a developer dependency? 🤔

Reply:
This is what npm install did. Would you recommend putting it in the devDependencies?

Reply:
Yeah, npm install puts it in the regular dependencies but with -D or --save-dev it puts the addOn to the devDependencies.
If you ask me personally, I would put everything into the devs, that is not important for running the app itself  - but there might be different approaches and some addOns even NEED to be installed without dev. 
Just wanted to know, if there was a special thought behind. 🤓